.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+    -   id: black
```

Comment:
we won't backport the `black` releated part of this and won't globally change src formatting in 1.2 branch.